vivisect/parsers/macho.py

```diff
@@ -1,8 +1,22 @@
 import os
+import struct
 
 import vivisect.parsers as viv_parsers
 import vstruct.defs.macho as vs_macho
-import vivisect.analysis.i386 as viv_a_i386
+
+macho_magics = (
+    vs_macho.MH_MAGIC,
+    vs_macho.MH_CIGAM,
+    vs_macho.MH_MAGIC_64,
+    vs_macho.MH_CIGAM_64,
+    vs_macho.FAT_MAGIC,
+    vs_macho.FAT_CIGAM,
+)
+
+def isParser(bytez):
+    bytemagic = struct.unpack('<I', bytez[:4])[0]
+    if bytemagic in macho_magics:
+        return 'macho'
```

Comment:
seems like this should return a bool (like other `isParser` routines). right now, it returns either a `str` or `None`.

Reply:
good catch.